graphs_source/import_diagram1.mermaid

```diff
@@ -0,0 +1,11 @@
+graph TD
```

Comment:
This literally looks like graphviz. Maybe we should switch to it, as @juanjux mentioned? We already have some internal tools to dump UASTs in graphviz format, so it sounds like a logical step.

Reply:
Yep. Mermaid was used initially because it was easier to integrate with JS server and client side as well as with the old gitbook plugins. Now that we generate them upfront, whatever you choose for sources is ok.